assets/js/atomic/blocks/product-elements/rating/block.json

```
@@ -0,0 +1,43 @@
+{
+	"name": "woocommerce/product-rating",
+	"version": "1.0.0",
+	"icon": "info",
+	"title": "Product Rating",
+	"description": "Display the average rating of a product.",
+	"attributes": {
+		"productId": {
+			"type": "number",
+			"default": 0
+		},
+		"isDescendentOfQueryLoop": {
+			"type": "boolean",
+			"default": false
+		},
+		"textAlign": {
+			"type": "string",
+			"default": ""
+		},
+		"isDescendentOfSingleProductBlock": {
+			"type": "boolean",
+			"default": false
+		},
+		"isDescendentOfSingleProductTemplate": {
+			"type": "boolean",
+			"default": true
```

In the other blocks, we set this value as default. I would say to be consistent.

```suggestion
			"default": false
```